docs/static/upgrading.asciidoc

```diff
@@ -66,12 +66,19 @@ Before upgrading Logstash, remember to read the <<breaking-changes,breaking chan
 If you are installing Logstash with other components in the Elastic Stack, also see the
 {stack-ref}/index.html[Elastic Stack installation and upgrade documentation].
 
-If you are using the default mapping templates in Logstash, to continue using these after migrating Elasticsearch to 6.0, you must override the existing template with the 6.x template.
- This can be done by starting a pipeline with the `overwrite_template => true` option in the Elasticsearch output definition in the Logstash config.
+==== Logstash 6.0.0 `document_type` Handling Bug
 
-Note that multiple doctypes are no longer supported in Elasticsearch 6.0. Please refer to
- {ref}/removal-of-types.html[Removal of mapping types] and {ref}/breaking-changes.html[Breaking changes] for more information.
+We’d like to alert users to a bug in Logstash 6.0.0 that can cause errors writing to Elasticsearch 6.0+ clusters. When the `type` field is set to more than one value per Elasticsearch index Logstash will encounter errors while indexing documents. Logstash 6.0 uses the value of the `type` field to set the Elasticsearch `type` by default. Elasticsearch 6 [no longer supports more than one type]((https://www.elastic.co/guide/en/elasticsearch/reference/current/removal-of-types.html) per index. In Logstash 6.0.1 we will fix this bug by changing the behavior so that when communicating with an Elasticsearch 6.0+ cluster Logstash will not set the document type by default, but rather, will set it to the value `doc`, regardless of whether an event’s `[type]` field has been set. The type information of a document will still be available under the field `type`.
```

Comment:
This should be more focused on the scenario that triggers the problem:
Writing events that have more than one value for the `type` field into the same index of an ES 6.x cluster.
Because by default we fetch the document_type from this `type` field, when writing to ES 6.x you must disable this behaviour by setting the type explicitly to a single value like `document_type => "log"`.